Chat/TokenManagement/IUserTokenManager.cs

```diff
@@ -7,5 +7,6 @@ namespace Chat
 	public interface IUserTokenManager
     {
         Task<(string userMri, string skypeToken, long expiresDateUtc)> GenerateTokenAsync(string resourceConnectionString);
+        Task<(string userMri, string skypeToken, long expiresDateUtc)> RefreshTokenAsync(string resourceConnectionString, string expiredToken);
```

Comment:
```suggestion
        Task<(string userIdentity, string token, long expiresDateUtc)> RefreshTokenAsync(string resourceConnectionString, string expiredToken);
```

Reply:
making sure we dont say mri/skype in this change.. I think there are many other occurences. For now I will say those are out of scope of this change